fields/main.go

```diff
@@ -306,6 +306,24 @@ func main() {
 				f.OmitEmpty = true
 				return nil
 			}
+		case "SettingMgmt":
+			sshKeyField := NewFieldInfo(resource.StructName+"XSshKeys", "x_ssh_keys", "struct", "", false, false)
+			sshKeyField.Fields = map[string]*FieldInfo{
+				"name":        NewFieldInfo("name", "name", "string", "", false, false),
+				"keyType":     NewFieldInfo("keyType", "type", "string", "", false, false),
```

Comment:
I named this `keyType` instead of `type` since the latter is a reserved keyword for Go